tests/playwright/tests/Subscription/BuyPro.spec.ts

```diff
@@ -0,0 +1,77 @@
+  await expect(page.locator('img').nth(3)).toBeVisible();
+  await expect(page.locator('#react-portal-modal-container svg').nth(2)).toBeVisible();
+  await expect(page.getByText('Multi-Send', { exact: true })).toBeVisible();
+  await expect(page.getByText('With multi-send feature you')).toBeVisible();
+  await expect(page.locator('#react-portal-modal-container')).toContainText('With multi-send feature you can send funds up to 255 addresses at once. Tonkeeper Pro saves time and simplifies the process of mass transactions.');
+  await expect(page.locator('img').nth(4)).toBeVisible();
+  await expect(page.locator('#react-portal-modal-container')).toContainText('And other cool features are on the way.');
+  await expect(page.getByRole('button', { name: 'Try Pro for Free' }).nth(1)).toBeVisible();
+  await expect(page.getByText('Try Pro for FreeBuy Pro').nth(1)).toBeVisible();
+  await expect(page.locator('#react-portal-modal-container').getByRole('button').first()).toBeVisible();
+  await page.getByRole('button', { name: 'Buy Pro' }).first().click();
+  await expect(page.getByText('Tonkeeper ProTonkeeper Pro\'s')).toBeVisible();
+  await expect(page.locator('#react-portal-modal-container')).toContainText('Tonkeeper ProTonkeeper Pro\'s subscription comes with an extended wallet feature, offering a toolset for crypto management.');
+  await expect(page.getByText('Select Wallet for')).toBeVisible();
+  await expect(page.locator('#react-portal-modal-container')).toContainText('Select Wallet for authorization');
+  await expect(page.locator('div').filter({ hasText: /^trust wallet - 12 words mnemonic🤖UQBC…_BrRW5$/ }).nth(1)).toBeVisible();
+  await expect(page.locator('div').filter({ hasText: /^trust wallet - 12 words mnemonic🤖UQDe…xygmv4R2$/ }).nth(1)).toBeVisible();
+  await page.locator('#react-portal-modal-container').getByText('UQBC…_BrRW5').click();
+  await expect(page.getByText('Enter password')).toBeVisible();
+  await expect(page.getByRole('button', { name: 'Cancel' })).toBeVisible();
+  await page.getByLabel('Password').click();
+  await page.getByLabel('Password').fill('123456');
+  await page.getByText('PasswordCancelConfirm').click();
+  await page.getByRole('button', { name: 'Confirm' }).click();
+  await expect(page.getByRole('heading', { name: 'Tonkeeper Pro' })).toBeVisible();
+  await expect(page.getByText('Tonkeeper Pro\'s subscription')).toBeVisible();
+  await expect(page.locator('#react-portal-modal-container')).toContainText('Tonkeeper Pro\'s subscription comes with an extended wallet feature, offering a toolset for crypto management.');
+  await expect(page.locator('div').filter({ hasText: /^trust wallet - 12 words mnemonic🤖UQBC…_BrRW5$/ }).nth(2)).toBeVisible();
+  await expect(page.getByText('Yearly Pro Package')).toBeVisible();
+  await expect(page.getByText('8.00 TON')).toBeVisible();
```

Comment:
Price could be changed, for my perspective better to not check the amount

Better to check that, list of plan's is not empty